cmd/backup/script.go

```diff
@@ -73,6 +74,15 @@ func newScript(c *Config) *script {
 }
 
 func (s *script) init() error {
+	if s.c != nil && s.c.BackupJitter > 0 {
+		max := s.c.BackupJitter
+		delay := time.Duration(rand.Int63n(int64(max) + 1))
+		if delay > 0 {
+			s.logger.Info("applying startup jitter", "delay", delay)
+			time.Sleep(delay)
+		}
+	}
+
```

Comment:
The logic  in here is fine, but I find it a bit surprising to have the `init` function itself sleep in certain cases. Could you move this a level up to `runScript` where the entire flow of procedures is orchestrated?